webapp/components/search_bar.jsx

```diff
@@ -270,15 +274,6 @@ export default class SearchBar extends React.Component {
                 >
                     <span className='fa fa-angle-left'/>
                 </div>
-                <span
-                    className='search__clear'
-                    onClick={this.clearFocus}
-                >
-                    <FormattedMessage
-                        id='search_bar.cancel'
```

Comment:
it should be fine if you remove it from `en.json` the process for translations will get rid of the other ones for you